gits/terminal.py

```diff
@@ -71,25 +70,13 @@ def __init__(self, rows=24, cols=80):
             self.new_sci_seq_re[k.replace('\\E', '\x1b')] = v
 
         self.new_sci_seq_re_compiled = []
-        self.csi_seq = {
-            '`': (self._cap_kb2, [1]),
-        }
         for k, v in list(self.new_sci_seq_re.items()):
             res = k.replace('[', '\['). \
```

Comment:
@dshil, yeah, but I don't want to touch the lines without any strong reason because they are not connected with the current changes.